src/features/analytics/AnalyticsPage.tsx

```diff
@@ -18,27 +29,33 @@ type CellTable<TypeProps> = {
   }
 };
 
-type FooterTable<TypeProps> = {
-  filteredRows: Array<{
-    values: TypeProps;
-  }>;
-};
-
 function AnalyticsPage() {
   const [isLoading, setIsLoading] = useState(true);
-  const [employees, setEmployees] = useState<AnalyticsSalaryForSeo[]>([]);
+  const [employees, setEmployees] = useState<(GetPreviewType)[]>([]);
+  const [total, setTotal] = useState<TotalFinance>();
 
   useEffect(() => {
+    loadTotals();
     loadEmployeesAsync();
   }, []);
 
+  function getSum(key: string) {
+    const sum = employees.map((el) => el[key as keyof GetPreviewType] as number)
+      .reduce((pre: number, current: number) => pre += (current || 0), 0);
+
+    return sum;
+  }
+
   return (
     <ContentCard
       isStickyHead
       headerContent={(
         <DefaultCardHeader>Analytics</DefaultCardHeader>
       )}
     >
+      <div className="analitycs-page--btns">
```

Comment:
please read about BEM ( I sent the link from the skype ) 
usually the name for classes is written in full